docs/user-guide/getting-started/install.md

```diff
@@ -173,25 +173,24 @@ pip3 install certora-cli-beta
 You can then switch to the standard CVL release by running `deactivate`, and
 back to the beta release using `certora-beta/bin/activate`.
 
-Step 3: Set the premium access key as an environment variable
+Step 3: Set the personal access key as an environment variable
 -------------------------------------------------------------
 
-The Certora Prover is available for all to use in a free trial version. The
-Prover's processing power and storage capacity are limited in the trial
-version. If you are using the tool in this way, you can skip to the next step.
+The Certora Prover requires a personal access key. 
+You can get a personal access key by registering on the 
```

Comment:
Is it free?  if so, this is still worth mentioning

Reply:
yes it is free, added